src/main/java/core/basesyntax/data/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.data.dao;
+
+import core.basesyntax.data.model.FruitTransaction;
+import core.basesyntax.data.model.OperationType;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String INPUT_DATA = "fruit_shop_transactions.csv";
```

Comment:
The `INPUT_DATA` constant is declared but never used in this class. Consider removing it if it is not needed to avoid confusion.